torch/special/__init__.py

```diff
@@ -159,6 +159,37 @@
     tensor([-0.9466,  2.6352,  0.6131, -1.7169,  0.6261])
 """.format(**common_args))
 
+logsumexp = _add_docstr(_special.special_logsumexp,
```

Comment:
Update the documentaion of `torch.logsumexp` to point to `torch.special.logsumexp`

Eg. 
https://github.com/pytorch/pytorch/pull/55878/files#diff-547193dc33a84fcae0a5f7d9d4096c298f2bb718132e70a9cf5ec90bdfb113aaL4441

Reply:
Done, thanks for pointing this out, Kshiteej!